### ISSUE TITLE
llama3 ollama version errors

### ISSUE THREAD

**Issue by Sjdavis25** [n4ze3m/dialoqbase#277]
I've seen online that ollama works when using Gemma2 but nothing mething any ollama version of llama3. I want to create a chatbot that is based off of my local version of llama3 but every time I upload my ollama version of llama3 no matter what the embedded method is I can never recive a proper response. Whenever I try and use the playground chat I receive the message "There was an error processing your request" and whether or not I have data sources uploaded  does not seem to affect this issue either.

**Comment by edwhiskey** [comment]
Did your problem solved? I tried Gemma2 and it doesn't work as well.

**Comment by Sjdavis25** [comment]
No, unfortunately I'm still having the same problem, still waiting for some sort of response

**Comment by n4ze3m** [comment]
Hey, sorry for the late reply. Can you share the logs, if possible?

**Comment by Sjdavis25** [comment]
![db logs](https://github.com/user-attachments/assets/4ac61231-0ce9-43b3-836f-791a558a6796)

**Comment by n4ze3m** [comment]
Are you using admin/api/chat or the playground (UI)

**Comment by Sjdavis25** [comment]
I'm using the playground UI

**Comment by edwhiskey** [comment]
![20240807](https://github.com/user-attachments/assets/a8494c51-d604-4128-bf8a-40875e220d0e)
my problem solved. You should pick the model which is uploaded not the one you named.
For me, I unseen all the unuse mode (e.g. from Open AI or Github), so I can get a clear list from Ollama.

**Comment by Sjdavis25** [comment]
When I try to pick a model I oly get the ones I named, am I uploading my models wrong?
![dialoqbaseSC](https://github.com/user-attachments/assets/fe2b91f0-14e8-4998-8213-000999caee5b)

**Comment by n4ze3m** [comment]
Hey, delete the current model, then go to Admin > Application, and turn on the fetch Ollama model dynamically

**Comment by Sjdavis25** [comment]
Got it to work thank you!!
